config.py

```diff
@@ -1 +1,16 @@
-DEBUG = True
+import os
+
+
+class Config(object):
+    DEBUG = False
+    PG_PASS = 'EazdJhQW4j'
```

Comment:
Fine for now cos we're just messing about trying to make something but we need to find the proper way to do this in a secure way.

Reply:
Totally, My understanding is the pass will eventually get stored as an env variable, but I also don't want to deal with the hassle this early on.